plugins/storage/volume/storpool/src/main/java/org/apache/cloudstack/storage/datastore/util/StorPoolUtil.java

```diff
@@ -517,6 +517,12 @@ public static SpApiResponse volumeUpdate(final String name, final Long newSize,
         return POST("MultiCluster/VolumeUpdate/" + name, json, conn);
     }
 
+    public static SpApiResponse volumeUpadateTags(String name, Map<String, String> tags, SpConnectionDesc conn) {
```

Comment:
```suggestion
    public static SpApiResponse volumeRemoveTags(String name, SpConnectionDesc conn) {
            Map<String, String> tags = StorPoolHelper.addStorPoolTags(null, "", null, "");
```

seems this method is used to remove tags

Reply:
Thanks @sureshanaparti! I prefer it to stay with this name. In future, this method could be used to set tags also

Reply:
in the future `this method could be used to set tags also` but the name is `volumeRemoveTags`, @slavkap ? Remove and set seem in contradiction to me. Can we add/rename whatever we need in the future later?

Reply:
@slavkap `volumeUpadateTags` has a spello.
also
When we need a thing in the future we can implement it then, right?

Reply:
@DaanHoogland, I will rename it as you both suggested. Still, I prefer to leave it with a common name like the used StorPool API call even though now it is used only to remove

Reply:
that seems ok, but the spello is still strange: volumeUpadateTags instead of volumeUpdateTags?

Reply:
I renamed the method with the latest commit as Suresh suggested to volumeRemoveTags

Reply:
:shrug: